docs/inflight_exit_scenarios.md

```diff
@@ -0,0 +1,409 @@
+`/inflight_exit.get_data` 
+```json
+ {
+    "txbytes": "F3170101C0940000..."
+ }
+ ```
+
+ response:
+```json
+ {
+    "data": {
+        "txbytes": "F3170101C0940000...",
+        "sigs": "7C29FB8327F60BBFC62...",
+        "input_txs" : [
+            "F81891018080808...",
+            "2A0341808602A01..."
+        ],
+        "input_proofs" : [
+             "CEDB8B31D1E4C...",
+             "A67131D1E904C..."
+        ]
+    }
+ }
+ ```
+
+#### 2. Start the IFE
+```
+RootChain.startInFlightExit(
+    response.data.txbytes, 
+    response.data.input_txs, 
+    response.data.input_proofs, 
+    response.data.sigs, 
+    {"value": inFlightExitBond}
+)
+```
+
+#### 3. Piggyback his output
+The second argument is `4` because he is piggybacking the first output.
+```
+RootChain.piggybackInFlightExit(
+    response.data.txbytes,
+    4, 
+    {"value": piggybackBond}
+)
+```
+
+After finalization, if nobody challenges the exit, Bob will exit his output and get his `inFlightExitBond` and `piggybackBond` back.
+
+# Simple In-flight Exit with 2 outputs
+Alice sends tokens to Bob, but this time she includes a change output to herself, so that the transaction `tx1` has 2 outputs. 
+
+When Alice calls `watcher/status` she gets this response:
+
+```json
+{
+    "version": "1",
+    "success": true,
+    "data": {
+        "last_validated_child_block_number": 10000,
+        "last_mined_child_block_timestamp": 1535031020,
+        "last_mined_child_block_number": 11000,
+        "eth_syncing": true,
+        "byzantine_events": [
+            {
+                "event": "block_withholding",
+                "details": {
+                    "blockhash"  : "DB32876CC6F...",
+                    "blocknum"  : 10000,
+                }
+            },
+            {
+                "event": "piggyback_available",
+                "details": {
+                    "txbytes": "F3170101C0940000...",
+                    "available_outputs" : [1]
+                }
+            },
+        ],
+        "inflight_txs": [
+            {
+                "txhash": "230C450180808080...",
+                "txbytes": "F3170101C0940000...",
+                "input_addresses": [
+                    "0x1234..."
+                ],
+                "output_addresses": [
+                    "0x7890...",
+                    "0x1234..."
+                ],
+            }
+        ],
+        "inflight_exits": [
+            {
+                "txhash": "230C450180808080...",
+                "txbytes": "F3170101C0940000...",
+                "timestamp" : 1544193137,
+                "piggybacked_outputs" : [0]
+            }
+        ]
+    }
+}
+ ```
+
+Because Bob has already started an exit for the transaction - there is a `piggyback_available` event indicating that `output[1]` can be piggybacked. This output is to Alice's address. Note that as Alice is the sole owner of the inputs, she does not need to piggyback any input.
+
+#### 1. Piggyback her output on the IFE
```

Comment:
again, aren't we hitting the API to get Piggyback info? (either let's document that step, or maybe drop an unnecessary endpoint, leaving only `status`? not sure...)

Reply:
Yeah, that endpoint has been dropped - the info required is minimal (`txbytes` and `outputIndex`) and it's already in the event.